sys/suit/transport/coap.c

```diff
@@ -366,6 +366,7 @@ static void _suit_handle_url(const char *url)
         }
 
 #endif
+#ifdef MODULE_SUIT_STORAGE_FLASHWRITE
```

Comment:
Can IS_ACTIVE be used?

Reply:
Not really, the `riotboot/slot.h` is also required in this block and that doesn't work on native, missing `SLOT0_LEN` etc.

Reply:
We could 

```C
#define SLOT0_LEN   (FLASHPAGE_SIZE * FLASHPAGE_NUMOF) / 2
```

Reply:
I'd rather refactor the flashwrite code from this file. While currently the main application of SUIT in RIOT is limited to flashwrite, this is not the only area where SUIT is applicable and I'd rather not start hard requiring flashwrite (or riotboot) to use the coap SUIT transport.